main/src/ca/uwaterloo/flix/language/phase/jvm/JvmName.scala

```diff
@@ -122,6 +122,16 @@ object JvmName {
     */
   val Exception: JvmName = JvmName(List("java", "lang"), "Exception")
 
+  /**
+   * The `java.lang.RuntimeException` name
+   */
+  val RuntimeException: JvmName = JvmName(List("java", "lang"), "RuntimeException")
+
+  /**
+   * The FlixError name
+   */
+  val FlixError: JvmName = JvmName(Nil, "FlixError")
```

Comment:
I think we should put these things into `dev.flix.runtime.FlixError` and `dev.flix.runtime.X` in general.

This avoids a situation where some creates an enum called `FlixError` and havoc ensures.

(As for Unit it makes sense of it to stay in the root).